packages/shared/components/inputs/RecipientInput.svelte

```diff
@@ -33,9 +32,11 @@
     }
 
     $: hasFocus && (error = '')
-    $: hasFocus && modal?.open()
     $: value && modal?.open()
-    $: keepRecipientAccountSelectorOpen = hasFocus
+
+    $: if (hasFocus) {
+        setTimeout(() => modal.open(), 101)
```

Comment:
This is used because the Modals have a 100ms lock on getting re-opened.

Reply:
Im getting an error here 🙏🏼 
```bash
TypeError: Cannot read property 'open' of undefined
    at RecipientInput.svelte:27
```
```suggestion
        setTimeout(() => modal?.open(), 101)
```

Reply:
it only happens in profiles that only have 1 wallet